static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-sections.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+const sections = ['politics', 'world', 'business', 'commentisfree'];
+
+const getSection = function(section: string): string {
```

Comment:
Couldn't this have been converted to an arrow function? eg.

`const getSection = (section: string): string => {//do something here};`